oak/common/policy.cc

```diff
@@ -16,9 +16,35 @@
 
 #include "oak/common/policy.h"
 
+#include "absl/strings/escaping.h"
+
 namespace oak {
 
-const char kOakLabelGrpcMetadataKey[] = "x-oak-label";
+const char kOakPolicyGrpcMetadataKey[] = "x-oak-policy";
 
 const char kOakAuthorizationBearerTokenGrpcMetadataKey[] = "x-oak-authorization-bearer-token";
+
+std::string SerializePolicy(const oak::policy::Labels& policy_proto) {
```

Comment:
This is doing something slightly different to what it says on the tin: it's serializing (converting a `struct` to a blob o' data) but it's also encoding that data in a text-friendly format (base64).  

(I suspect these should be two separate steps: use binary blob / protobuf `bytes` in general, and drop to base64 only at the places where the value needs to go over the wire in a text-friendly format.)